uber/models.py

```diff
@@ -297,9 +297,11 @@ def _ints(self, name, val):
     @suffix_property
     def _label(self, name, val):
         try:
-            return self.get_field(name).type.choices[int(val)]
-        except KeyError:
-            return ''
+            val = int(val)
+        except ValueError:
+            return val
+
+        return self.get_field(name).type.choices.get(val)
```

Comment:
This fixes the rare case of a value in a dictionary not having the label we expect it to have. This is happening right now for single-days badges, though it should technically never happen.